docs/docs/general/02-Getting Started/writing-your-first-test.md

```diff
@@ -28,11 +46,19 @@ npx jest test/my-first-askui-test-suite.test.ts --config ./test/jest.config.ts
 
 You should now see the test suite being executed inside the shell and, actually, your cursor should move to some text shown on your screen and click on that text. :tada: Congratulations! You just executed your first test suite using askui.
 
+### Troubleshooting
+
+If you are having problems with the test execution, [have a look at our Troubleshooting page](../07-Troubleshooting/index.md)
+
 ## Manual
 
 At the beginning we create a new folder with the name `test`. For the next step we create the `my-first-askui-test-suite.test.ts` file in our `test` folder. It is also possible to create test files in other folders and with other names.
 
-Copy the following over into that file:
+We need you to create some credentials through our [askui user portal](https://app.askui.com/) (usage is free!).
+Please see <a href="../askui%20User%20Portal/signup" target="_blank">our documentation on how to signup and create the credentials</a>.
+
+Copy the following over into that file.
+Do not forget to replace your workspace Id and your access token in the _UiControlClient_:
```

Comment:
I would lowercase id and actually I think I would put _worspace id_ and _access token_ it into ``` so that it is formatted as code and name it exactly like in code so that relation becomes clear.

Reply:
Great idea!